cloudify_agent/resources/pm/initd/initd.conf.template

```diff
@@ -30,7 +29,7 @@ CELERYD_LOG_LEVEL={{ log_level }}
 CELERYD_PID_FILE="{{ pid_file }}"
 CELERYD_LOG_FILE="{{ log_file }}"
 CELERYD_ENV_DIR={{ virtualenv_path }}
-CELERYD_OPTS="--events -Q {{ queue }} --hostname={{ name }} --autoscale={{ max_workers }},{{ min_workers }} --maxtasksperchild=10 -Ofair --without-gossip --without-mingle"
+CELERYD_OPTS="--events -Q {{ queue }} --hostname={{ name }} --autoscale={{ max_workers }},{{ min_workers }} --maxtasksperchild=10 -Ofair --without-gossip --without-mingle --config=cloudify.broker_conf"
```

Comment:
Celery broker configuration- specifically, this one reads from the broker_conf config file that you've commented on elsewhere.

This allows:
a) Broker SSL configuration in a sane manner (it can be exceedingly ugly to configure from exported vars).
b) Brokers that don't display the username+password (which reside in the URL) when anyone on the system runs ps.

Well, (b) would be true if I'd actually removed the hostname from this one. *making note*